File: Practice/30 Days of Code/Day 8 Dictionaries and Maps/Day 8 Dictionaries and Maps.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
N = int(input())
phoneBook = {}
for _ in range(N):
    name, phoneNumber = input().split()
    phoneBook[name] = phoneNumber
while True:
    try:
        query = input()
    except:
        break
    if query in phoneBook:
        print( query + "=" + phoneBook[query] )
    else:
        print( "Not found" )

File: Practice/30 Days of Code/Day 8 Dictionaries and Maps/Day 8 Dictionaries and Maps.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$n = rtrim(fgets($_fp));
$phoneBook = [];
for($i=0; $i<$n; $i++) {
    list($k,$v) = explode(' ',rtrim(fgets($_fp)));
    $phoneBook[$k] = $v;
}

while( $k = rtrim(fgets($_fp)) ) {
    if( !array_key_exists($k,$phoneBook) ) {
        echo "Not found\n";
        continue;
    }
    echo $k.'='.$phoneBook[$k]."\n";
}

?>
